File: 236a_bog.py
from sys import stdin, stdout


def input():
    return stdin.readline().strip()


def print(string):
    return stdout.write(str(string) + "\n")


def main():
    print("IGNORE HIM!" if len(set(input())) % 2 else "CHAT WITH HER!")


if __name__ == "__main__":
    main()

File: 236a_bog.cpp
#include <bits/stdc++.h>

int main()
{
	using namespace std;
	ios::sync_with_stdio(0);
	cin.tie(0);
	string s;
	cin >> s;
	set<char> st(s.begin(), s.end());
	st.size() % 2 ? cout << "IGNORE HIM!\n" : cout << "CHAT WITH HER!\n";
}
